### DISCUSSION TITLE
connection with database

### DISCUSSION THREAD

**Discussion by Fiatlux11** [hasura/graphql-engine#8662] [Question] [answered]
Hi, i want to connect my Wikidatabase with GraphQL in Hasura, i use the DataBase-URL but I get every time the same error message; "Adding data source failed Inconsistent object: mssql connection error" how can i solve this? why couldnt i connect it with my database?

**Comment by meetzaveri** [comment, marked as answer]
@Fiatlux11 Hello

Can I know which platform you are using for your development ?

1. Hasura CE (open source version)?
2. Hasura Cloud ?

Also, we have some users reporting MSSQL connection issues with mac m1 system (which is because MSSQL docker image hasn't released a version that supports `arm64` arch which mac m1 system deals with). 

So can we know that which system are you using for your development ?

Thanks.

**Reply by Fiatlux11** [reply]
I use hasura cloud and my database wiki base, I have also and sparql api.

**Reply by meetzaveri** [reply]
Hello,

We would recommend creating a ticket under "Cloud Account Support" in "Help & support" section from your Hasura cloud dashboard. As you are using hasura cloud and because **we would need your project details to troubleshoot the problem**, it would be suitable to have a private discussion (as opposed to github discussion) via creating a ticket under "Cloud Account Support" which wouldn't expose the details in public as github discussions are publicly accessible.

I hope you understood the situation.

Thanks!
